src/app-bar.jsx

```diff
@@ -314,26 +313,26 @@ const AppBar = React.createClass({
           {children}
       </Paper>
     );
-  },
+  }
 
+  @autobind
```

Comment:
react-router seems to follow this convention too (https://github.com/rackt/react-router/blob/master/examples/huge-apps/components/GlobalNav.js).
I don't know why they and facebook do so, but I feel like we can trust them.